com.unity.render-pipelines.universal/Runtime/ShadowUtils.cs

```diff
@@ -57,6 +57,9 @@ public static bool ExtractDirectionalLightMatrix(ref CullingResults cullResults,
             shadowSliceData.resolution = shadowResolution;
             shadowSliceData.shadowTransform = GetShadowTransform(shadowSliceData.projectionMatrix, shadowSliceData.viewMatrix);
 
+            // This used to be fixed to .6f, but is now configureable.
```

Comment:
Perhaps add to the comment that this is "culling sphere radius multiplier", so it's clear to the next reader. I had do a search to understand that as "**cascade blend culling** factor" seems like something "completely" different.

Reply:
sure. Also in the end i changed the value to 0.6 because I think i makes more sense, since it was the original value